.github/workflows/publish.yaml

```diff
@@ -0,0 +1,55 @@
+name : Publish Images
+
+on:
+  push:
+    tags:
+      - "*"
+
+env:
+  REGISTRY: docker.io
+  REPO : rancher
+
+
+jobs:
+  ci :
+    uses: rancher/backup-restore-operator/.github/workflows/ci.yaml@release/v5.0
```

Comment:
Decided to leave this as reference to the current default branch, but open to changing it to reference the target branch `release/v4.0` -- applies to a couple of other places.

Applies to a couple of other places. Perhaps discussing/documenting backport rules for CI is a good idea to be addressed in a follow up PR?

Reply:
I'm ok with this. If some future version is no longer compatible with the version in `release/v5.0` we can redefine it in that branch, and then have future branches reference that one.
For example, say we have to redefine the CI body in `release/v6.0`. Then the line in `release/7.0` etc. would be `uses: rancher/backup-restore-operator/.github/workflows/ci.yaml@release/v6.0`

So then if we want to be consistent, maybe this CI body should be defined in `release/v3.0' and referenced in `v4` and `v5`? WDYT?

Reply:
Yeah that sounds like a good idea, `release/v3.0` is probably the best bet, but we could also consider referencing `master`

Another idea is to have an orphan branch that specifically holds the CI files... Although that could complicate things more than help, but we could design around that to make it easier to work with